src/main/kotlin/com/atlassian/performance/tools/report/api/judge/ActionReport.kt

```diff
@@ -0,0 +1,16 @@
+package com.atlassian.performance.tools.report.api.judge
+
+import com.atlassian.performance.tools.jiraactions.api.ActionType
+import com.atlassian.performance.tools.report.api.junit.JUnitReport
+
+/**
+ * Action aware report
+ *
+ * @property action Action associated with this report.
+ * @property nonExceptional True if the failure in this report is expected sometimes as a part of a normal flow, false otherwise.
+ */
+interface ActionReport : JUnitReport {
```

Comment:
Why do we couple JPT-specific stuff like VU actions with JUnit stuff like XML reports?
This leads to implementation sniffing down the line.
Why is `Verdict.failedActions` insufficient?

Reply:
It is coupled already, because verdict relies on JUnitReport (there are reports for actions but anonymous). 
We've added ActionReport to classify the reports properly.